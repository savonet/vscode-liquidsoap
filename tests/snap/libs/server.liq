# Enable telnet server.
# @category Interaction
# @param ~port Port on which we should listen.
def server.telnet(~port=1234) =
  settings.server.telnet.port := port
  settings.server.telnet := true
end
