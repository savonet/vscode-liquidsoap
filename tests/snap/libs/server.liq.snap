># Enable telnet server.
#^ source.liquidsoap comment.line.number-sign.liquidsoap
# ^^^^^^^^^^^^^^^^^^^^^^ source.liquidsoap comment.line.number-sign.liquidsoap
># @category Interaction
#^ source.liquidsoap comment.line.number-sign.liquidsoap
# ^ source.liquidsoap comment.line.number-sign.liquidsoap
#  ^^^^^^^^^ source.liquidsoap comment.line.number-sign.liquidsoap comment.doc.category.liquidsoap
#           ^^^^^^^^^^^^ source.liquidsoap comment.line.number-sign.liquidsoap
># @param ~port Port on which we should listen.
#^ source.liquidsoap comment.line.number-sign.liquidsoap
# ^ source.liquidsoap comment.line.number-sign.liquidsoap
#  ^^^^^^ source.liquidsoap comment.line.number-sign.liquidsoap comment.doc.param.liquidsoap
#        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.liquidsoap comment.line.number-sign.liquidsoap
>def server.telnet(~port=1234) =
#^^^ source.liquidsoap keyword.other.function.definition.liquidsoap
#   ^ source.liquidsoap
#    ^^^^^^^^^^^^^ source.liquidsoap entity.name.function.binding.liquidsoap
#                 ^ source.liquidsoap
#                  ^ source.liquidsoap
#                   ^^^^ source.liquidsoap variable.parameter.liquidsoap
#                       ^ source.liquidsoap
#                        ^^^^ source.liquidsoap constant.numeric.decimal.integer.liquidsoap
#                            ^ source.liquidsoap
#                             ^^ source.liquidsoap
>  settings.server.telnet.port := port
#^^ source.liquidsoap
#  ^^^^^^^^ source.liquidsoap variable.liquidsoap
#          ^ source.liquidsoap keyword.other.liquidsoap
#           ^^^^^^ source.liquidsoap variable.method.liquidsoap
#                 ^ source.liquidsoap keyword.other.liquidsoap
#                  ^^^^^^ source.liquidsoap variable.method.liquidsoap
#                        ^ source.liquidsoap keyword.other.liquidsoap
#                         ^^^^ source.liquidsoap variable.method.liquidsoap
#                             ^ source.liquidsoap
#                              ^^ source.liquidsoap keyword.operator.set.liquidsoap
#                                ^ source.liquidsoap
#                                 ^^^^ source.liquidsoap variable.liquidsoap
>  settings.server.telnet := true
#^^ source.liquidsoap
#  ^^^^^^^^ source.liquidsoap variable.liquidsoap
#          ^ source.liquidsoap keyword.other.liquidsoap
#           ^^^^^^ source.liquidsoap variable.method.liquidsoap
#                 ^ source.liquidsoap keyword.other.liquidsoap
#                  ^^^^^^ source.liquidsoap variable.method.liquidsoap
#                        ^ source.liquidsoap
#                         ^^ source.liquidsoap keyword.operator.set.liquidsoap
#                           ^ source.liquidsoap
#                            ^^^^ source.liquidsoap constant.language.boolean.liquidsoap
>end
#^^^ source.liquidsoap keyword.other.function.end.liquidsoap
>